.nais/naiserator-dev.yaml

```diff
@@ -0,0 +1,179 @@
+          namespace: teamsykefravr
+          cluster: dev-gcp
+        - application: finnfastlege
+          namespace: teamsykefravr
+          cluster: dev-gcp
+        - application: syfobehandlendeenhet
+          namespace: teamsykefravr
+          cluster: dev-gcp
+        - application: syfomodiaperson
+          namespace: teamsykefravr
+          cluster: dev-gcp
+        - application: syfooversiktsrv
+          namespace: teamsykefravr
+          cluster: dev-gcp
+        - application: syfoperson
+          namespace: teamsykefravr
+          cluster: dev-gcp
+        - application: esyfovarsel
+          namespace: team-esyfo
+          cluster: dev-gcp
+        - application: syfomotebehov
+          namespace: team-esyfo
+          cluster: dev-fss
+        - application: syfooppfolgingsplanservice
+          namespace: team-esyfo
+          cluster: dev-fss
+        - application: spinnsyn-backend
+          namespace: flex
+          cluster: dev-gcp
+        - application: sykepengesoknad-backend
+          namespace: flex
+          cluster: dev-gcp
+        - application: smregistrering-backend
+          namespace: teamsykmelding
+          cluster: dev-gcp
+        - application: syfosmmanuell-backend
+          namespace: teamsykmelding
+          cluster: dev-gcp
+        - application: syfosmregister
+          namespace: teamsykmelding
+          cluster: dev-gcp
+        - application: syk-dig-backend
+          namespace: teamsykmelding
+          cluster: dev-gcp
+    outbound:
+      external:
+        - host: "axsys.dev-fss-pub.nais.io"
+        - host: "norg2.dev-fss-pub.nais.io"
+        - host: "graph.microsoft.com"
+        - host: "pdl-api.dev-fss-pub.nais.io"
+        - host: "audit.nais"
+          ports:
+            - name: audit
+              port: 6514
+              protocol: TCP
+      rules:
+        - application: syfobehandlendeenhet
+        - application: syfo-tilgangskontroll-redis
+        - application: skjermede-personer-pip
+          namespace: nom
+  azure:
+    application:
+      allowAllUsers: true
+      enabled: true
+      tenant: trygdeetaten.no
+      claims:
+        extra:
+          - "NAVident"
+  env:
+    - name: AXSYS_URL
+      value: "https://axsys.dev-fss-pub.nais.io"
+    - name: GRAPHAPI_URL
+      value: "https://graph.microsoft.com"
+    - name: NORG2_URL
+      value: "https://norg2.dev-fss-pub.nais.io"
+    - name: PDL_CLIENT_ID
+      value: "dev-fss.pdl.pdl-api"
+    - name: PDL_URL
+      value: "https://pdl-api.dev-fss-pub.nais.io/graphql"
+    - name: SYFOBEHANDLENDEENHET_CLIENT_ID
+      value: "dev-gcp.teamsykefravr.syfobehandlendeenhet"
+    - name: SYFOBEHANDLENDEENHET_URL
+      value: "http://syfobehandlendeenhet"
+    - name: SKJERMEDE_PERSONER_CLIENT_ID
+      value: "dev-gcp.nom.skjermede-personer-pip"
+    - name: SKJERMEDE_PERSONER_URL
+      value: "http://skjermede-personer-pip.nom"
+
+    - name: OLD_ROLE_KODE6_ID
```

Comment:
Burde vi droppe disse gamle kodene i ny app? (Må uansett fjernes om noen måneder - https://trello.com/c/6os35hB8/1435-frist-for-%C3%A5-fjerne-gamle-tilgangskontroller-innen-1september)

Reply:
Vi ble enige på produktprat om at de må beholdes litt til